sdk/keyvault/azure-security-keyvault-secrets/src/test/java/com/azure/security/keyvault/secrets/SecretClientTestBase.java

```diff
@@ -45,6 +45,9 @@
     private static final String SECRET_NAME = "javaSecretTemp";
     private static final String SECRET_VALUE = "Chocolate is hidden in the toothpaste cabinet";
 
+    private static final String SDK_NAME = "client_name";
```

Comment:
All the other places I see this constant called "name" and "version". Does this work with "client_name"?

Reply:
KeyVault tried to use real version and name to run their tests. If we tried to retrieve the real one, based on our structure, we have to read the property again. Since it is not required in their tests, so I make the fake one. 

Moreover, I have written another test class to test UserAgentProperties specifically.